services/movie_session.py

```diff
@@ -0,0 +1,48 @@
+import datetime
+from django.db.models import QuerySet
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int) -> None:
+
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: datetime.date = None) -> QuerySet:
+    movies_sessions = MovieSession.objects.all()
+
+    if session_date:
+        movies_sessions = movies_sessions.filter(show_time__date=session_date)
+    return movies_sessions
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
```

Comment:
Consider handling the `ObjectDoesNotExist` exception when retrieving a movie session by ID. This will prevent the application from crashing if the session ID does not exist.